src/AstGen.zig

```diff
@@ -214,7 +214,20 @@ pub fn deinit(astgen: *AstGen, gpa: Allocator) void {
     astgen.ref_table.deinit(gpa);
 }
 
-pub const ResultLoc = union(enum) {
+pub const ResultLocTag = enum {
```

Comment:
```suggestion
pub const ResultLocTag = std.meta.Tag(ResultLoc);
```
I don't want this kind of repetition.